services/client/src/api/pools/useFetchHypePools.tsx

```diff
@@ -7,16 +7,22 @@ const hypePoolsPerPage = 6;
 interface FetchHypesFilter {
   first: number;
   skip: number;
+  orderBy: string;
```

Comment:
here it might make sense to limit the choices to the fields that make sense as a type: 
```suggestion
  orderBy: 'endDate' | 'startDate' | ... ;
```